src/main/java/com/minecrafttas/common/Configuration.java

```diff
@@ -48,7 +48,7 @@ public Properties load() {
 			e.printStackTrace();
 			return null;
 		} catch (FileNotFoundException e) {
-			e.printStackTrace();
+			System.out.println("No config file found: "+file);
 			return null;
```

Comment:
logging?

Reply:
Plan is to make a cutom logger for the common package...